sentry-apollo/src/main/java/io/sentry/apollo/SentryApolloInterceptor.kt

```diff
@@ -39,7 +39,12 @@ class SentryApolloInterceptor(
             val sentryTraceHeader = span.toSentryTrace()
 
             // we have no access to URI, no way to verify tracing origins
-            val headers = request.requestHeaders.toBuilder().addHeader(sentryTraceHeader.name, sentryTraceHeader.value).build()
+            val requestHeaderBuilder = request.requestHeaders.toBuilder()
```

Comment:
Should we check for `TracingOrigins.contain(hub.options.tracingOrigins, request.url.toString())` here as well? We do in other places.

Reply:
But the comment above says there's no way to verify them?

Reply:
Ah, I missed that here, thanks.